.chloggen/move-operator-crd-install-method.yaml

```
@@ -1,12 +1,15 @@
 # One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
-change_type: bug_fix
+change_type: breaking
 # The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
 component: operator
 # A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
 note: Move operator CRD installation to the crds/ folder via a subchart to resolve Helm install ordering issues
 # One or more tracking issues related to the change
-issues: [1561]
+issues: [1561,1619]
 # (Optional) One or more lines of additional information to render under the primary note.
 # These lines will be padded with 2 spaces and then inserted directly into the document.
 # Use pipe (|) for multiline entries.
-subtext:
+subtext: |
+  - Users enabling the operator (`.Values.operator.enabled=true`) must now set `operatorcrds.install=true` in Helm values or [manually manage CRD installation](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/auto-instrumentation-install.md#crd-management).
+  - Previously, CRDs were installed using templates (`operator.crds.create=true`), which could cause race conditions and installation failures.
+  - CRD installation is now handled via Helm's native `crds/` directory for better stability, using a [localized subchart](https://github.com/signalfx/splunk-otel-collector-chart/tree/main/helm-charts/splunk-otel-collector/charts/opentelemetry-operator-crds).
```

```suggestion
  - Users must set `operatorcrds.install=true` along with the existing option `operator.enabled=true` to install CRDs when deploying the opentelemetry-operator from this chart version onwards.
  - CRD installation is now handled via Helm's native `crds/` directory for better stability, using a [localized subchart](https://github.com/signalfx/splunk-otel-collector-chart/tree/main/helm-charts/splunk-otel-collector/charts/opentelemetry-operator-crds). Refer to the [helm guide](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you) for further information about the new approach.
  - Users trying to upgrade to this chart version with the operator already enabled (`operator.enabled=true`) can keep the installed CRDs by setting the arg `operator.crds.create=true`. The chart sets this to `false` by default in this version. This deprecated approach installs CRDs using templates which could cause race conditions and installation failures. Users are encouraged to reinstall with the new CRDs installation approach.
```
Maybe something like this will work. Assuming we are removing the schema change for operator.crds.create.

---

Added a link to https://github.com/signalfx/splunk-otel-collector-chart/blob/main/UPGRADING.md#0113-to-0116 which will cover this info.